app/models/edition.rb

```diff
@@ -6,6 +6,8 @@ class Edition < ApplicationRecord
   has_many :participations, dependent: :destroy
   has_many :participants, through: :participations
 
+  has_many :participations_with_points, -> { with_points }, class_name: 'Participation', inverse_of: :edition
+
```

Comment:
any reason we use so many empty lines between **has_many**?

Reply:
I'd say that's a matter of taste here, nothing wrong for Rubocop